kong/plugins/influxdb-log/handler.lua

```diff
@@ -0,0 +1,105 @@
+local HttpLogHandler = require "kong.plugins.http-log.handler"
+
+local InfluxdbLogHandler = HttpLogHandler:extend()
```

Comment:
@thefosk  Sorry, I may not understand what you mean.   @thibaultcha I agree with you.

so cool,  this is easy to understand and use